modules/jooby-rxjava/src/main/java/org/jooby/rx/Rx.java

```diff
@@ -366,7 +366,7 @@ public DeferredSubscriber(final Deferred deferred) {
     @Override
     public void onCompleted() {
       if (done.compareAndSet(false, true)) {
-        deferred.resolve((Object) null);
+        deferred.resolve(null);
```

Comment:
Are you sure about this? Feel we need the cast to `Object`

Reply:
You're right about `resolve(Object)`, please ignore my comment.

Are you on linux? I did see that error in Travis, still can't reproduce/fix it

Reply:
no problem :-) Happy to help!

Nope, I'm on macOS (so kinda posix yeah). I found this regarding the error:

https://stackoverflow.com/questions/12190377/getting-javassist-notfoundexception-with-powermock-and-powerrule-in-junit-with-m

maybe we just need to switch to the agent version? I'm not that deep into your testing (yet), but maybe this helps?

Reply:
btw: locally I can't reproduce this error neither. All tests (maven/console and IntelliJ) are fine.